code/__DEFINES/atmospherics/atmos_mapping_helpers.dm

```diff
@@ -1,40 +1,46 @@
 //OPEN TURF ATMOS
 /// the default air mix that open turfs spawn
-#define OPENTURF_DEFAULT_ATMOS "o2=22;n2=82;TEMP=293.15"
-#define OPENTURF_LOW_PRESSURE "o2=14;n2=30;TEMP=293.15"
+#define OPENTURF_DEFAULT_ATMOS "[GAS_O2]=22;[GAS_N2]=82;TEMP=[T20C]"
+#define OPENTURF_LOW_PRESSURE "[GAS_O2]=14;[GAS_N2]=30;TEMP=[T20C]"
 /// -193,15°C telecommunications. also used for xenobiology slime killrooms
-#define TCOMMS_ATMOS "n2=100;TEMP=80"
+#define TCOMMS_ATMOS "[GAS_N2]=100;TEMP=80"
 /// space
-#define AIRLESS_ATMOS "TEMP=2.7"
+#define AIRLESS_ATMOS "TEMP=[TCMB]"
 /// -93.15°C snow and ice turfs
-#define FROZEN_ATMOS "o2=22;n2=82;TEMP=180"
+#define FROZEN_ATMOS "[GAS_O2]=22;[GAS_N2]=82;TEMP=[FROZEN_ROOM_TEMP]"
+/// -14°C snow and ice turfs, a more breatheable coldroom atmos.
+#define COLD_ATMOS "[GAS_O2]=22;[GAS_N2]=82;TEMP=[COLD_ROOM_TEMP]"
 /// -14°C kitchen coldroom, just might loss your tail; higher amount of mol to reach about 101.3 kpA
-#define KITCHEN_COLDROOM_ATMOS "o2=26;n2=97;TEMP=[COLD_ROOM_TEMP]"
+#define KITCHEN_COLDROOM_ATMOS "[GAS_O2]=26;[GAS_N2]=97;TEMP=[COLD_ROOM_TEMP]"
 /// used in the holodeck burn test program
-#define BURNMIX_ATMOS "o2=2500;plasma=5000;TEMP=370"
+#define BURNMIX_ATMOS "[GAS_O2]=2500;[GAS_PLASMA]=5000;TEMP=370"
+///-153.15°C plasma air, used for burning people.
+#define BURNING_COLD "[GAS_N2]=82;[GAS_PLASMA]=24;TEMP=120"
+///Space temperature hyper nob
+#define SPACE_TEMP_NOBLIUM "[GAS_HYPER_NOBLIUM]=7500;TEMP=[TCMB]"
```

Comment:
I'm not sure this is worthwhile.

Reply:
Just like the whole pr I guess, moving these away from being compile time doesn't seem all that worth it.
You could do something similar to this by making the defines text objects, and use + instead of []
that way you get almost the same result, but it can be compile time

Reply:
It would look pretty weird unless I have the strings defined as well, or maybe that won't actually matter much since these are defines so it would probably be fine/even better this way.
![image](https://github.com/tgstation/tgstation/assets/53777086/8405d9ab-ad27-4a70-8404-7fcdb07dd412)

Reply:
I made temperature a define as well

![image](https://github.com/tgstation/tgstation/assets/53777086/bbc067c3-5429-4041-abb0-1af77177d7c3)